src/components/EditPofileModal/EditProfileModal.jsx

```diff
@@ -1,38 +1,84 @@
-import React from "react";
+import React, { useEffect, useRef, useState } from "react";
+import { useDispatch } from "react-redux";
+import { updateUser } from "redux/reducers/usersSlice";
 
-const EditProfileModal = ({ ...params }) => {
-  const { setShowEditProfileModal } = params;
+const EditProfileModal = ({ setShowEditProfileModal, currentProfileUser }) => {
+  const { username, name, profile_pic, bio, website } = currentProfileUser;
+
+  const [selectedImage, setSelectedImage] = useState();
+  const [imgUrl, setImgUrl] = useState(profile_pic);
+
+  const nameRef = useRef();
+  const usernameRef = useRef();
+  const bioRef = useRef();
+  const websiteRef = useRef();
+
+  const dispatch = useDispatch();
+
+  const fileChangeHandler = (e) => {
+    setSelectedImage(e.target.files[0]);
+  };
+
+  const uploadImage = async (e) => {
+    e.preventDefault();
+    const formData = new FormData();
+    formData.append("file", selectedImage);
+    formData.append("upload_preset", process.env.REACT_APP_UPLOAD_PRESET);
+
+    fetch("https://api.cloudinary.com/v1_1/dl0nhw7w3/image/upload", {
+      method: "POST",
+      body: formData,
+    })
+      .then((response) => {
+        return response.json();
+      })
```

Comment:
```suggestion
      .then((response) =>  response.json())
```

this may also work but ur code is ok